src/saving/InProgressSave.cs

```diff
@@ -166,11 +168,22 @@ private void Step()
             case State.Initial:
                 // On this frame a pause menu might still be open, wait until next frame for it to close before
                 // taking a screenshot
+                wasColourblindScreenFilterVisible = ColourblindScreenFilter.Instance.Visible;
+                if (wasColourblindScreenFilterVisible)
+                {
+                    ColourblindScreenFilter.Instance.Hide();
+                }
+
                 state = State.Screenshot;
                 break;
             case State.Screenshot:
             {
                 save = createSaveData.Invoke(this);
+                if (wasColourblindScreenFilterVisible)
```

Comment:
Please add a blank line before this to separate out that this is not related to the `save` variable initialization.